.github/workflows/build.sh

```diff
@@ -0,0 +1,32 @@
+if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then
```

Comment:
looks like this `build.sh` shell script is no longer used after you switched from muiltibuild to cibuildwheel